packages/fxa-settings/src/components/Tooltip/index.tsx

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { ReactNode } from 'react';
+import classNames from 'classnames';
+
+export type TooltipType = 'default' | 'error';
+export type PositionType = 'top' | 'bottom';
+
+type TooltipProps = {
+  message: string;
+  type?: TooltipType;
+  anchorLeft?: boolean;
+  className?: string;
+  position?: PositionType;
+};
+
+export const Tooltip = ({
+  message,
+  className,
+  type = 'default',
+  anchorLeft = false,
+  position = 'top',
+}: TooltipProps) => {
+  const bgColor = type === 'error' ? 'red-600' : 'grey-500';
+  // caret position and the positioning rule for tooltip are actually the opposite
+  // of the passed in position;
+  const oppositePositionRule = {
+    top: 'bottom',
+    bottom: 'top',
+  }[position];
+
+  return (
+    <div
```

Comment:
I think we're going to have trouble with this when it comes to narrow anchors. For example, we have this button to take a photo that will need to display a tooltip:

<img width="98" alt="Screen Shot 2020-09-22 at 11 50 52 AM" src="https://user-images.githubusercontent.com/6392049/93906219-ef33d900-fcc9-11ea-8e13-b7fdf99a5a84.png">

But when I try to apply the tooltip to a narrow element it becomes off-center:

<img width="112" alt="Screen Shot 2020-09-22 at 11 51 58 AM" src="https://user-images.githubusercontent.com/6392049/93906353-15f20f80-fcca-11ea-9dc0-9ae0d5981824.png">

Granted, some of this may be up to the specific use-case to supply custom classes to the component to, say, force the width of tooltip, but generally I think it should maintain its center positioning.

Reply:
I don't really know what to do to address this, would the `left-1/2 transform -translate-x-1/2` rule solve this?